src/integrations/action-handlers/ngpvan-action.js

```diff
@@ -62,7 +62,7 @@ export const postCanvassResponse = async (contact, organization, body) => {
   // eslint-disable-next-line no-console
   console.info("Sending contact update to VAN", {
     vanId,
-    body
```

Comment:
Note to reviewers: this is an improvement to logging for the ngpvan action handler.